markdown/org/grading.md

```diff
@@ -7,4 +7,233 @@ hidden: true
+einmal Abzüge bekommen, können Sie dies in einer anderen Woche nacharbeiten (unter Beachtung der
+Maximal-15-Punkte-pro-Woche-Regel).
+
+Jede der beiden Aufgaben aus Pool P geht über zwei Wochen und bringt maximal 20 Punkte. Diese Aufgaben
+sind nicht mit Aufgaben aus Pool A/B/C kombinierbar; eine Nacharbeitung ist hier nicht möglich. Sie
+können hier frühestens in der Woche nach dem E-Assessment starten.
+
+Parallel zu Pool A/B/C/P bearbeiten Sie jede Woche eine vorgegebene Aufgabe aus Pool K für je 10 Punkte.
+
+#### Praktische Teilnote
+:::
+
+*   220P Gesamt, 40P Puffer => “Krankheitsregelung”, 180P == 100%
+*   4.0: ab 60% v. 180P (108P), alle 4% nächste Teilnote, 1.0: ab 96% v. 180P (172,8P)
+
+::: notes
+Sie können maximal in den Pools A+B+C+P 120 Punkte und in Pool K 100 Punkte erreichen, in Summe
+also 220 Punkte. Die 100% für die Notenberechnung wird auf 180 Punkte festgelegt. Damit haben Sie
+genügend Schlupf, falls Sie einmal erkranken oder wegen anderer familiärer Angelegenheiten nicht
+zur Vorstellung Ihrer Lösung im Praktikum erscheinen können.
+
+**Achtung**: Sie müssen im Laufe des Semesters in **jedem Pool** die jeweils angegebene
+Mindestanzahl der Punkte zum Bestehen erreichen, sonst haben Sie die praktische Teilleistung
+insgesamt nicht bestanden.
+
+
+Beachten Sie zusätzlich die "Hinweise zum Praktikum", die sich auf die Inhalte und die formalen
+Aspekte Ihrer Abgaben beziehen.
+:::
+
+
+### 2. Theoretische Teilleistung
+
+*   Erster Prüfungszeitraum: 2x digitale Prüfung: 1x im Semester ("E-Assessment") plus 1x in erster Prüfungsphase
+*   Zweiter Prüfungszeitraum: _Eine_ digitale Klausur
+
+::: notes
+Die Prüfung für die theoretische Teilleistung wird als digitale Klausur über eine
+spezielle ILIAS-Instanz ("E-Assessment-Server") durchgeführt.
+
+Im ersten Prüfungszeitraum besteht die theoretische Prüfung aus zwei kürzeren Teilprüfungen
+("E-Assessments") mit je 45 Minuten Dauer, die einmal etwa in der Mitte der Vorlesungszeit
+und einmal in der ersten Prüfungsphase angeboten werden. Die Note ergibt sich dabei aus dem
+Mittelwert der beiden Teilprüfungen.
+
+Im zweiten Prüfungszeitraum besteht die theoretische Prüfung aus einer digitalen Klausur mit
+90 Minuten Dauer.
+
+Es wird jeweils rechtzeitig bekanntgegeben, ob die Prüfungen wie bisher im Home-Office durchgeführt
+werden oder ob dafür Rechner der FH genutzt werden, d.h. eine Anwesenheit in den Räumen der FH am
+Campus Minden notwendig ist. Die Termine in den Prüfungszeiträumen werden rechtzeitig vom Prüfungsamt
+bekannt gegeben, für den Termin in der Vorlesungszeit siehe `[Fahrplan]({{< ref "/org/schedule" >}})`{=markdown}.
+:::
+
+
+### 3. Gesamtnote: Gewichteter Mittelwert der beiden Teilleistungen
+
+**60% Praxis, 40% Theorie**
+
+[[Hinweis Abgabeslots, Präsenz/Online]{.bsp}]{.slides}
+
+
+## Hinweise zum Praktikum
+
+### Bearbeitung, Teams
+
+Sie bearbeiten alle Aufgaben in festen Teams zu je **zwei Personen**. Jedes Team erarbeitet seine
```

Comment:
Zwei Personen? Wir haben nie drüber gesprochen, aber sonst hatten wir doch immer dreier Teams.

Reply:
hmmm, wir haben 16 personen pro praktikum, d.h. bei 2er teams wären das 8 gruppen, bei 3er teams 6 gruppen. 

2er teams: wenn wir pro termin noch einen tutor haben, könnten jonas + tutor in den 90 minuten 4 teams parallel "bearbeiten", d.h. mit lüften und so ca. 20 minuten pro team. das finde ich eigentlich recht großzügig.

alternativ wären das bei 3er teams dann eher 30 minuten zeit je team.

@AMatutat was denkst du?

Reply:
Ich bin für dreier Teams. Es ist ja nicht nur die Abgabezeit (wobei ich 20min auch ehr für zu wenig halte) sondern auch die Bearbeitungszeit. Und grade wenn wir wollen, dass die Studis über das Minimum hinausgehen, sollten wir sie nicht mit dem Minimum Zeitlich auslasten.

Reply:
ok, ist ein argument 👍

Reply:
done.